### PR TITLE
[v7r2] returning integer out of new-style division

### PR DESCRIPTION
**Pull request by fstagni**
Several systems: adapting to new style division.

BEGINRELEASENOTES

*Subsystem
NEW/CHANGE/FIX: explanation

ENDRELEASENOTES

### CODE REVIEW

File: Core/Utilities/ExecutorDispatcher.py [hunk 1]
```diff
@@ -605,7 +605,7 @@ def __getNextExecutor(self, taskId):
     return result
 
   def getTaskIds(self):
-    return self.__tasks.keys()
+    return list(self.__tasks.keys())
```
Comment (andresailer) [inline Core/Utilities/ExecutorDispatcher.py:608]:
```suggestion
    return list(self.__tasks)
```

File: Core/Utilities/Glue2.py [hunk 2]
```diff
@@ -134,9 +134,10 @@ def __getGlue2ShareInfo(host, shareInfoLists):
       ceInfo['Queues'] = {}
       queueInfo = {}
       queueInfo['GlueCEStateStatus'] = shareInfoDict['GLUE2ComputingShareServingState']
-      queueInfo['GlueCEPolicyMaxCPUTime'] = str(int(shareInfoDict.get('GLUE2ComputingShareMaxCPUTime', 86400)) / 60)
-      queueInfo['GlueCEPolicyMaxWallClockTime'] = str(int(shareInfoDict
-                                                          .get('GLUE2ComputingShareMaxWallTime', 86400)) / 60)
+      queueInfo['GlueCEPolicyMaxCPUTime'] = str(int(int(
```
Comment (chrisburr) [inline Core/Utilities/Glue2.py:137]: This is just adding `int(` to an existing `int(`?

Comment (andresailer) [reply Core/Utilities/Glue2.py:137]: The output of the `shareInfoDict.get` could be a string, which needs to be casted, no magic cast to default value.

File: Core/Utilities/ExecutorDispatcher.py (hunk 1 above)
Comment (fstagni) [reply Core/Utilities/ExecutorDispatcher.py]: Done.

File: Core/Utilities/Glue2.py (hunk 2 above)
Comment (fstagni) [reply Core/Utilities/Glue2.py:137]: There's 
```
int(shareInfoDict.get('GLUE2ComputingShareMaxWallTime', 86400))
```
and then that value is divided by 60: the new `int` cast is for that division.

Comment (chrisburr) [reply Core/Utilities/Glue2.py:137]: Ah I see it now, sorry failing to pair brackets correctly...

**Review by atsareg — APPROVED** [review]

**Pull request merged**